### PR TITLE
Update README and versions for 2.47.0 / 24.06

### PR DESCRIPTION
**Pull request by mc-nv**
#### Checklist
- [x] PR title reflects the change and is of format `<commit_type>: <Title>`
- [ ] Changes are described in the pull request.
- [ ] Related issues are referenced.
- [ ] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [ ] Added [test plan](#test-plan) and verified test passes.
- [ ] Verified that the PR passes existing CI.
- [ ] Verified copyright is correct on all changed files.
- [ ] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [ ] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
Check the [conventional commit type](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
box here and add the label to the github PR.
- [x] build
- [x] ci
- [x] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] revert
- [ ] style
- [ ] test

#### Test plan:

- CI Pipeline ID:

#### Related Issues: (use one of the action keywords Closes / Fixes / Resolves / Relates to)
- closes GitHub issue: #xxx

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -69,10 +69,10 @@
...
 #
 TRITON_VERSION_MAP = {
-    "2.47.0dev": (
-        "24.06dev",  # triton container
-        "24.05",  # upstream container
-        "1.18.0",  # ORT
+    "2.47.0": (
+        "24.06",  # triton container
+        "24.06",  # upstream container
+        "1.18.1",  # ORT
```
Comment (mc-nv) [inline build.py:75]: Expecting patch version...

File: qa/common/cuda_op_kernel.cu.cc.patch [hunk 2]
```diff
@@ -11,7 +11,7 @@ index a9d66f9..a92e218 100644
 -#include "tensorflow/core/util/gpu_kernel_helper.h"
 -#include "tensorflow/core/util/gpu_launch_config.h"
 +//#define EIGEN_USE_GPU
-+//#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
++//#include "unsupported/Eigen/CXX11/Tensor"
```
Comment (mc-nv) [inline qa/common/cuda_op_kernel.cu.cc.patch:14]: Dependency path changed inside the container.

**Review by rmccorm4 — DISMISSED** [review]
LGTM, but assuming we need to confirm the ORT version based on the comment: https://github.com/triton-inference-server/server/pull/7334#discussion_r1631815922

File: build.py (hunk 1 above)
Comment (mc-nv) [reply build.py]: Had to revert version to be able pass build.

**Review by nvda-mesharma — APPROVED** [review]
LGTM

**Comment by rmccorm4** [comment]
Going to merge this to simplify running release pipelines

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**